### PR TITLE
Changes for Open Cybersecurity Alliance migration

### PR DESCRIPTION
**Pull request by delliott90**
*** The below is required to be filled by any non-IBM employee, in order for their pull request to be accepted ***
DCO 1.1 Signed-off-by: [NAME] <[EMAIL]>

### CODE REVIEW

**Comment by delliott90** [comment]
@JasonKeirstead I'm wondering if we still need to have the Apache license in its own license file or if it's enough that we link to it from our readme files?

**Comment by JasonKeirstead** [comment]
It still should be in its own LICENSE file

**Comment by delliott90** [comment]
Does the CONTRIBUTING.md page need to be changed? It currently outlines the Developer's Certificate of Origin.

**Comment by JasonKeirstead** [comment]
Need to follow https://github.com/oasis-open-projects/documentation/tree/master/templates

File: setup.py [hunk 1]
```diff
@@ -73,7 +73,7 @@
     #
     # This field corresponds to the "Home-Page" metadata field:
     # https://packaging.python.org/specifications/core-metadata/#home-page-optional
-    url='https://github.com/IBM/stix-shifter',  # Optional
+    url='https://github.com/opencybersecurityalliance/stix-shifter',  # Optional
```
Comment (mdazam1942) [inline setup.py]: Should we also update this License stated at line 102?
`'License :: OSI Approved :: MIT License',`

**Pull request merged**